salt/utils/cloud.py

```diff
@@ -483,9 +510,6 @@ def bootstrap(vm_, opts):
         transport=opts.get('transport', 'zeromq')
     )
 
-    deployed = False
-    inline_script_deployed = False
```

Comment:
Looks like you'll need to leave the `inline_script_deployed` line in, but removing the `deployed` line is okay.